MapboxCoreNavigation/RouteController.swift

```diff
@@ -408,8 +430,12 @@ extension RouteController {
     func willReroute(notification: NSNotification) {
         _ = enqueueRerouteEvent()
     }
-
+    
     func didReroute(notification: NSNotification) {
+        if let _ = notification.userInfo?[FasterRouteFoundEvent] as? Bool {
+            _ = enqueueFoundFasterRouteEvent()
+        }
+        
         if let lastReroute = outstandingFeedbackEvents.map({$0 as? RerouteEvent }).last {
             lastReroute?.update(newRoute: routeProgress.route)
```

Comment:
This also needs to be called for faster routes found.

Reply:
Since `didReroute` is called after finding a faster route, won't this code path be hit?